cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/SubscriptionPatchDTO.java

```diff
@@ -84,7 +84,7 @@ public int getUnit()
 
     public boolean hasValidUpgradeData()
     {
-        return billingItem != null && value != null || ! billingItems.isEmpty();
+        return billingItem != null && value != null || billingItems != null && ! billingItems.isEmpty();
```

Comment:
this method can be put somewhere else in backend if it isn't of any use for the client?